blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_2.py

```diff
@@ -115,3 +127,15 @@ def _evaluate(diameter: MM) -> DIMENSIONLESS:
         if diameter <= 32:
             return 1
         return (132 - diameter) / 100
+
+    def latex(self) -> LatexFormula:
+        """Returns a LatexFormula object for this formula."""
+        numerical_equation = "1.00" if self.diameter <= 32 else f"(132 - {self.diameter}) / 100"
+
+        return LatexFormula(
+            return_symbol=r"\eta_2",
+            result=f"{self:.2f}",
+            equation=r"\begin{matrix} 1.0 & \text{voor }Ø ≤ 32 \\ (132 - Ø) / 100 & \text{voor }Ø > 32  \end{matrix}",
```

Comment:
shoulnd't we use english?

```suggestion
            equation=r"\begin{matrix} 1.0 & \text{for }Ø ≤ 32 \\ (132 - Ø) / 100 & \text{for }Ø > 32  \end{matrix}",
```